edb/edgeql/compiler/func.py

```diff
@@ -713,4 +756,9 @@ def finalize_args(
 
         args.append(call_arg)
 
+        # If we have any logged paths left over and our enclosing
+        # context is logging paths, propagate them up.
+        if arg_ctx and arg_ctx.path_log and ctx.path_log is not None:
```

Comment:
Should it be an error to have leftover paths in the log and enclosing context NOT logging paths?

Reply:
No, that happens all the time. For example in `SELECT Foo.num + 1`, we'll log `Foo.num`, we won't process it, but the enclosing context doesn't care.